vaadin-master-detail-layout-flow-parent/vaadin-master-detail-layout-flow/src/main/java/com/vaadin/flow/component/masterdetaillayout/MasterDetailLayout.java

```diff
@@ -263,6 +278,132 @@ public void setDetailMinSize(float minSize, Unit unit) {
                 HasSize.getCssSize(minSize, unit));
     }
 
+    /**
+     * Gets the orientation of the layout.
```

Comment:
We could mention the respective defaults in the new getters.

Reply:
Done.